### PR TITLE
Compile to JavaScript before publishing package

### PR DESCRIPTION
**Pull request by duvholt**

### CODE REVIEW

File: src/components/ExampleBox.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
```
Comment (oleast) [inline src/components/ExampleBox.tsx]: Why is this needed when we use 'allowSyntheticDefaultImports'?

Comment (duvholt) [reply src/components/ExampleBox.tsx]: Not really sure, but I figured it was easier to change it than fighting some silly warning from the build system.
